scripts/create_lightning_rc.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+OLDVER=0.37.0
+LVER=0.38.0
+rreplace(){
+   grep -rl "$1" . | xargs sed -i "s|$1|$2|g"
+}
+
+# clean up and checkout master
+git reset . && git checkout .
+git fetch
+git checkout master
+git pull
+
+# create local branch
+git branch -d v${LVER}_rc
+git checkout -b v${LVER}_rc
+
+# update lightning version
+sed -i "/$LVER/d" pennylane_lightning/core/_version.py
+echo '__version__ = "'${LVER}'-rc0"' >> pennylane_lightning/core/_version.py
+sed -i "s|Release ${LVER}-dev|Release ${LVER}|g" .github/CHANGELOG.md
+
+# commit & push lightning version
+git add -u pennylane_lightning/core/_version.py .github/CHANGELOG.md
+git commit -m "Create v${LVER} RC branch."
+git push --set-upstream origin v${LVER}_rc
+
+# update pennylane dep
+for file in requirements-dev.txt requirements-tests.txt; do
+    sed -i "s|pennylane.git@master|pennylane.git@v${LVER}-rc0|g" $file
+    git add -u $file
+done
+git commit -m "Target PennyLane v${LVER}-rc0 in requirements-tests.txt."
+git push 
+
+# upload wheel artifacts
+cd .github/workflows
+rreplace "          github.event_name == 'release'" "          github.event_name == 'pull_request'"
+cd -
+git add -u .github/workflows
+git commit -m "Change trigger to upload wheel artifacts."
```

Comment:
can we merge these two as follow:
```suggestion
git commit -am "Change trigger to upload wheel artifacts."
```